src/tests/regress/data/sql/arrays.sql

```diff
@@ -440,7 +440,7 @@ select array[];
 -- all of the following should be accepted
 select '{}'::text[];
 --@ select '{{{1,2,3,4},{2,3,4,5}},{{3,4,5,6},{4,5,6,7}}}'::text[];
---@ select '{0 second  ,0 second}'::interval[];
+select '{0 second  ,0 second}'::interval[];
```

Comment:
We do not support different `IntervalStyle`
https://www.postgresql.org/docs/16/runtime-config-client.html#GUC-INTERVALSTYLE

Reply:
But why the style is not the default "postgres" style at this place? 🤔